### PR TITLE
RI-2722-fix filtered events

### PR DESCRIPTION
**Pull request by AmirAllayarovSofteq**

### CODE REVIEW

File: redisinsight/ui/src/telemetry/telemetryUtils.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ const sendEventTelemetry = (payload: ITelemetrySendEvent) => {
   const isAnalyticsGranted = checkIsAnalyticsGranted()
   setAnonymousId(isAnalyticsGranted)
 
+  console.log(event)
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/telemetry/telemetryUtils.ts:57]: hello

File: redisinsight/ui/src/slices/hash.ts [hunk 2]
```diff
@@ -209,6 +210,9 @@ export function fetchHashFields(
       )
 
       if (isStatusSuccessful(status)) {
+        const matchValue = !isGlob(match, { strict: false })
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/slices/hash.ts:213]: Could you move this function to exported utils?

File: redisinsight/ui/src/slices/keys.ts [hunk 3]
```diff
@@ -390,6 +391,12 @@ export function fetchKeys(cursor: string, count: number, onSuccess?: () => void,
           })
         )
         if (!!type || !!match) {
+          let matchValue = '*'
+          if (match !== '*' && !!match) {
+            matchValue = !isGlob(match, { strict: false })
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/slices/keys.ts:396]: the same

File: redisinsight/ui/src/telemetry/telemetryUtils.ts (hunk 1 above)
Comment (AmirAllayarovSofteq) [reply redisinsight/ui/src/telemetry/telemetryUtils.ts:57]: My fault

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**